### PR TITLE
change k8s context switch background color

### LINKED ISSUE #8695: [UI] K8s Context Switcher: Dark Mode Color
### Current Behavior
The color of the K8s context switcher (the dropdown slider) matches that of the page's title bar.

### Desired Behavior
The color of the K8s context switcher (the dropdown slider) needs to be a shade slightly lighter than that of the page title background.

### Screenshots/Logs
Screenshot 2023-09-08 at 5 25 08 PM

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by AkashKumar7902**
**Notes for Reviewers**

This PR fixes #8695

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ghat0tkach** [comment]
hey @AkashKumar7902 

Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by AkashKumar7902** [comment]
@Ghat0tkach yeah sure !

**Comment by AkashKumar7902** [comment]
@Ghat0tkach sorry for the ping, I was wondering is there a google calender for the meetings ?

**Comment by leecalcote** [comment]
Thanks a bunch, @AkashKumar7902 👏

File: ui/components/Header.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ const styles = (theme) => ({
   headerIcons : { fontSize : "1.5rem", height : "1.5rem", width : "1.5rem" },
   cbadge : {
     fontSize : "0.65rem",
-    backgroundColor : "white",
+    backgroundColor : "#ffffffd4",
```
Comment (Yashsharma1911) [inline ui/components/Header.js:115]: is this right color @AkashKumar7902 🤔

Comment (AkashKumar7902) [reply ui/components/Header.js:115]: @Yashsharma1911 The color is slightly lighter than that of the page title background as asked in the issue description.

Comment (Yashsharma1911) [reply ui/components/Header.js]: Ohh Thanks for explaining I thought It was wrong

**Review by Yashsharma1911 — COMMENTED** [review]
@AkashKumar7902 does this PR make sure changes are reflected both on light and dark mode, if you can share screen shot it would be helpful, and let's then merge this PR

**Review by leecalcote — CHANGES_REQUESTED** [review]
@AkashKumar7902 This is not the desired affect:

The requested change is not with respect to the badge on the k8s icon. Will you be fixing this or given the signifi
Screenshot 2023-10-14 at 9 34 58 PM
Screenshot 2023-10-14 at 9 34 49 PM
cant misunderstanding, should we unassign and allow another contributor to address it?

File: ui/components/Header.js [hunk 2]
```diff
@@ -108,7 +108,7 @@ const styles = (theme) => ({
   headerIcons: { fontSize: '1.5rem', height: '1.5rem', width: '1.5rem' },
   cbadge: {
     fontSize: '0.65rem',
-    backgroundColor: 'white',
+    backgroundColor: 'ffffffd4',
```
Comment (leecalcote) [inline ui/components/Header.js:111]:
```suggestion
    backgroundColor: 'white',
```

**Pull request merged**